ci-operator/config/openshift/cloud-provider-nutanix/openshift-cloud-provider-nutanix-main.yaml

```diff
@@ -0,0 +1,53 @@
+base_images:
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.12
+binary_build_commands: go build .
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+canonical_go_repository: github.com/nutanix-cloud-native/cloud-provider-nutanix
+images:
+- dockerfile_path: openshift/Dockerfile.openshift
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12
+  to: nutanix-controller-manager
+promotion:
+  name: "4.12"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws
```

Comment:
Maybe we could run linter and unittests instead?

Don't think that it worth to run aws e2e there now...

Reply:
You need to have some sort of E2E before adding this to payload, is there a nutanix E2E?